internal/aws/cfn/cfn.go

```diff
@@ -261,7 +265,11 @@ func CreateChangeSet(template cft.Template, params []types.Parameter, tags map[s
 		config.Debugf("ChangeSet status: %s", status)
 
 		if status == "FAILED" {
-			return changeSetName, errors.New(ptr.ToString(res.StatusReason))
+			if ptr.ToString(res.StatusReason) == noChangeFoundMsg {
```

Comment:
I don't think this logic adds any value here. Would prefer moving this into the deploy command to decide behaviour there.

Reply:
Thank you for this feedback! I was just debating wheter I should add this logic into deploy.go or not. I'll modify it.